integration-cli/docker_utils.go

```diff
@@ -503,9 +503,12 @@ func init() {
 	// intelligent decisions about how to configure themselves, and validate
 	// that the target platform is valid.
 	res, b, err := sockRequestRaw("GET", "/version", nil, "application/json")
-	defer b.Close()
-	if err != nil || res.StatusCode != http.StatusOK {
-		panic("Init failed to get version: " + err.Error() + " " + string(res.StatusCode))
+	if b != nil {
+		defer b.Close()
+	}
+	if err != nil || res == nil || (res != nil && res.StatusCode != http.StatusOK) {
+		p := fmt.Sprintf("Init failed to get version: %s. Res=%v", err.Error(), res)
```

Comment:
no need to create the variable `p` you can inline, also: use `%v` and `err` it's more idiomatic